crates/node/service/src/actors/rpc.rs

```diff
@@ -37,40 +37,24 @@ pub enum RpcActorError {
 pub struct RpcActor {
     /// A launcher for the rpc.
     config: RpcBuilder,
-    /// The network rpc sender.
-    network: mpsc::Sender<P2pRpcRequest>,
-    /// The l1 watcher queries sender.
-    l1_watcher_queries: mpsc::Sender<L1WatcherQueries>,
-    /// The engine query sender.
-    engine_query: mpsc::Sender<EngineQueries>,
 }
 
 impl RpcActor {
     /// Constructs a new [`RpcActor`] given the [`RpcBuilder`].
-    pub fn new(config: RpcBuilder) -> (RpcOutboundData, Self) {
-        let (network_sender, network_recv) = mpsc::channel(1024);
-        let (l1_watcher_queries_sender, l1_watcher_queries_recv) = mpsc::channel(1024);
-        let (engine_query_sender, engine_query_recv) = mpsc::channel(1024);
-
-        (
-            RpcOutboundData {
-                network: network_recv,
-                l1_watcher_queries: l1_watcher_queries_recv,
-                engine_query: engine_query_recv,
-            },
-            Self {
-                config,
-                network: network_sender,
-                l1_watcher_queries: l1_watcher_queries_sender,
-                engine_query: engine_query_sender,
-            },
-        )
+    pub const fn new(config: RpcBuilder) -> ((), Self) {
+        ((), Self { config })
     }
```

Comment:
Can we just change this to
```suggestion
    pub const fn new(config: RpcBuilder) -> Self {
        Self { config }
    }
```